### DISCUSSION TITLE
iOS is throttling animations when Low Power Mode is enabled. Any workarounds?

### DISCUSSION THREAD

**Discussion by silasabbott** [motiondivision/motionone#255] [Support]
I am using Motion One for swipe action gestures and a few other UI interactions in my Capacitor app. In native iOS apps, swipe gestures are not throttled when Low Power Mode is enabled, but on the web iOS is throttling to ~30fps (probably throttling `requestAnimationFrame`).

This makes sense—animations on the web are likely heavier than native animations and should be throttled. With UI interactions however, it really degrades the user experience by making touch interactions and gestures choppy.

Is there a workaround for this now, or could this be addressed in a future version of Motion?

Ionic web apps also have this issue afaik.

**Comment by mattgperry** [comment]
Hey there - there sadly isn't a way around this! It's locked to 30fps in both WAAPI and rAF annoyingly
